src/assets/locales/es-MX/translation.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
Other than "Settings," it seems like the other variables aren't translated to Spanish?

Reply:
The translations are not completed, the support to add languages is.